Doc/extending/embedding.rst

```diff
@@ -275,6 +275,91 @@ will depend on the details of the C++ system used; in general you will need to
 write the main program in C++, and use the C++ compiler to compile and link your
 program.  There is no need to recompile Python itself using C++.
 
+.. _freezingandembeddingmanually:
+
+Freezing Modules in Manually Embedded Python
+============================================
+
+While it is possible to use frozen modules in embedded python if the main module
+is also frozen. It seems people are bitten when they don't have their main module
+frozen and use ``PyRun_SimpleString`` that contains their main module(s) but
+also depend on an frozen module (like the ``__hello__`` module). However if we
+wanted to add our own module to the frozen list we would normally do this::
+
+    #include <importlib.h>
+    #include <importlib_external.h>
+    #include "mymodule.h"
+
+    static const struct _frozen _PyImport_FrozenModules[] = {
+        /* importlib */
+        {"_frozen_importlib", _Py_M__importlib, (int)sizeof(_Py_M__importlib)},
+        {"_frozen_importlib_external", _Py_M__importlib_external,
+            (int)sizeof(_Py_M__importlib_external)},
+        /* mymodule */
+       {"mymodule", M_mymodule, (int)sizeof(M_mymodule)},
+       {0, 0, 0} /* sentinel */
+    };
+
+    const struct _frozen * PyImport_FrozenModules = _PyImport_FrozenModules;
+
+.. note::
+   The reason why this example includes ``importlib.h`` and
+   ``importlib_external.h`` is because of the fact that your embedded
+   interpreter might need them to run properly (load the python standard
```

Comment:
Generally with the rest of the page as well, I think less 'you/our' is good in a reference page, but I'm not an expert in that department.

```suggestion
   ``importlib_external.h`` is because of the fact that the embedded
   interpreter might need them to run properly (load the Python standard
```